### PR TITLE
Wip new

### PR DESCRIPTION
**Pull request by sullysaurus**
@jmburges @AnnJohn @SophieDeBenedetto

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,37 @@
...
+
+???
+
+# Basic Git Quiz
+
+?: Which of the following is the correct way to initialize a new Git repository?
+
+( )`git add .` (X)`git init` ( )`git commit`
+
+?: You can type `git status` at any point to check the status of your files.
```
Comment (jmburges) [inline README.md]: "You can type `git status` at any point while in a git controlled directory to check the status of your files"

File: README.md [hunk 2]
```diff
@@ -1 +1,37 @@
...
+
+(X)True
+( )False
+
+?: Which of the following commands will stage your entire directory?
```
Comment (jmburges) [inline README.md]: append "and every directory inside your current directory"

File: README.md [hunk 3]
```diff
@@ -1 +1,37 @@
...
+
+( )`git status all`
+(X)`git add .`
+( )`git commit all`
+
+?: We just created a new file called `index.html`, which of the following will stage this one file so we can commit it?
+
+(X)`git add index.html`
```
Comment (jmburges) [inline README.md:26]: add `git commit index.html` as an option

File: README.md [hunk 4]
```diff
@@ -1 +1,37 @@
...
+( )`git add new`
+
+
+?: Which of the following will commit `index.html` successfully?
+
+(X)git commit -m "new commit"
```
Comment (jmburges) [inline README.md]: backticks

Comment (jmburges) [inline README.md]: I wouldn't make this about commiting index.html. maybe give them something like:

`git status` returns: (git status output showing index is staged). How would you commit this with the message "adding new authors to index"

We also wanna give good examples of commit messages as possible!

Comment (jmburges) [inline README.md]: pretty much, you are never committing a file...you are committing whatever is staged. `add` is for adding a file to the staged area

**Comment by jmburges** [comment]
back at you

File: README.md (hunk 1 above)
Comment (sullysaurus) [inline README.md]: Updated.

File: README.md (hunk 2 above)
Comment (sullysaurus) [inline README.md]: Updated.

File: README.md (hunk 3 above)
Comment (sullysaurus) [inline README.md:26]: Updated.

**Pull request merged**